images/server/install-sambacc.sh

```diff
@@ -34,7 +34,7 @@ case $action in
         container_json_file="/usr/local/share/sambacc/examples/minimal.json"
     ;;
     install-rpm)
-        dnf install -y "${rpmfiles[0]}"
+        dnf install --skip-broken --nobes -y "${rpmfiles[0]}"
```

Comment:
Adding `--skip-broken` makes me think were asking to miss important error cases in the future. Unless you (plural) convince me it is necessary.

Reply:
> Adding `--skip-broken` makes me think were asking to miss important error cases in the future. Unless you (plural) convince me it is necessary.
@anoopcs9  has to do the convincing ... It came from him.